dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -0,0 +1,287 @@
+
+	insinto /usr/$(get_libdir)/${PN}/help
+	doins help/help.fdb
+
+	exeinto /usr/$(get_libdir)/firebird/intl
+	dolib.so intl/libfbintl.so
+	dosym "${D}"/usr/$(get_libdir)/libfbintl.so /usr/$(get_libdir)/${PN}/intl/fbintl
+	dosym "${D}"/etc/firebird/fbintl.conf /usr/$(get_libdir)/${PN}/intl/fbintl.conf
+
+	exeinto /usr/$(get_libdir)/${PN}/plugins
+	dolib.so plugins/libfbtrace.so
+	dosym "${D}"/usr/$(get_libdir)/libfbtrace.so /usr/$(get_libdir)/${PN}/plugins/libfbtrace.so
+
+	exeinto /usr/$(get_libdir)/${PN}/UDF
+	doexe UDF/*.so
+
+	insinto /usr/share/${PN}/upgrade
+	doins "${S}"/src/misc/upgrade/v2/*
+
+	insinto /etc/${PN}
+	insopts -m0644 -o firebird -g firebird
+	doins ../install/misc/*.conf
+	insopts -m0660 -o firebird -g firebird
+	doins security2.fdb
+
+	if use xinetd ; then
+		insinto /etc/xinetd.d
+		newins "${FILESDIR}/${PN}.xinetd" ${PN}
+	else
+		newinitd "${FILESDIR}/${PN}.init.d.2.5" ${PN}
+		newconfd "${FILESDIR}/${PN}.conf.d.2.5" ${PN}
+		fperms 640 /etc/conf.d/${PN}
+	fi
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}/${PN}.logrotate" ${PN}
+	fperms 0644 /etc/logrotate.d/${PN}
+
+	diropts -m 755 -o firebird -g firebird
+	dodir /var/log/${PN}
+	dodir /var/run/${PN}
+	keepdir /var/log/${PN}
+	keepdir /var/run/${PN}
+
+	use examples && docinto examples
+}
+
+pkg_postinst() {
+	use client && return
+
+	# Hack to fix ownership/perms
+	chown -fR firebird:firebird "${ROOT}/etc/${PN}" "${ROOT}/usr/$(get_libdir)/${PN}"
+	chmod 750 "${ROOT}/etc/${PN}"
+}
+
+pkg_config() {
+	use client && return
+
+	# if found /etc/security.gdb from previous install, backup, and restore as
+	# /etc/security2.fdb
+	if [ -f "${ROOT}/etc/firebird/security.gdb" ] ; then
+		# if we have scurity2.fdb already, back it 1st
+		if [ -f "${ROOT}/etc/firebird/security2.fdb" ] ; then
+			cp "${ROOT}/etc/firebird/security2.fdb" "${ROOT}/etc/firebird/security2.fdb.old"
+		fi
+		gbak -B "${ROOT}/etc/firebird/security.gdb" "${ROOT}/etc/firebird/security.gbk"
+		gbak -R "${ROOT}/etc/firebird/security.gbk" "${ROOT}/etc/firebird/security2.fdb"
+		mv "${ROOT}/etc/firebird/security.gdb" "${ROOT}/etc/firebird/security.gdb.old"
+		rm "${ROOT}/etc/firebird/security.gbk"
+
+		# make sure they are readable only to firebird
+		chown firebird:firebird "${ROOT}/etc/firebird/{security.*,security2.*}"
+		chmod 660 "${ROOT}/etc/firebird/{security.*,security2.*}"
+
+		echo
+		einfo "Converted old security.gdb to security2.fdb, security.gdb has been "
+		einfo "renamed to security.gdb.old. if you had previous security2.fdb, "
+		einfo "it's backed to security2.fdb.old (all under ${ROOT}/etc/firebird)."
+		echo
+	fi
+
+	# we need to enable local access to the server
+	if [ ! -f "${ROOT}/etc/hosts.equiv" ] ; then
+		touch "${ROOT}/etc/hosts.equiv"
+		chown root:0 "${ROOT}/etc/hosts.equiv"
+		chmod u=rw,go=r "${ROOT}/etc/hosts.equiv"
+	fi
+
+	# add 'localhost.localdomain' to the hosts.equiv file...
+	if [ grep -q 'localhost.localdomain$' "${ROOT}/etc/hosts.equiv" 2>/dev/null ] ; then
```

Comment:
I never added this line, not sure who did. Again this was a revision bump where I added 1 line to fix an issue with the logrotate file having incorrect permissions.